AWS X-Ray integration documentation and cleanup (#8601)
DataDog/documentation: content/en/serverless/installation/_index.md

### 2. Install the Datadog Forwarder

> Install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs. 

Reviewer: ```suggestion
Install the [Datadog Forwarder Lambda function][2], which is required to ingest AWS Lambda traces, enhanced metrics, custom metrics, and logs. 
```

> **Note**: Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation stack.

Reviewer: ```suggestion
**Note**: Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation Stack.
```
